docs/topics/Gateway.md

```diff
@@ -229,11 +229,18 @@ GUILDS (1 << 0)
   - CHANNEL_UPDATE
   - CHANNEL_DELETE
   - CHANNEL_PINS_UPDATE
+  - THREAD_CREATE
+  - THREAD_UPDATE
+  - THREAD_DELETE
+  - THREAD_LIST_SYNC
+  - THREAD_MEMBER_UPDATE
+  - THREAD_MEMBERS_UPDATE
 
 GUILD_MEMBERS (1 << 1)
   - GUILD_MEMBER_ADD
   - GUILD_MEMBER_UPDATE
   - GUILD_MEMBER_REMOVE
+  - THREAD_MEMBERS_UPDATE
```

Comment:
Just to make sure this isn't a typo, We can get `THREAD_MEMBERS_UPDATE` by specifying both `GUILDS` and `GUILD_MEMBERS`?

Reply:
Nope, only one is required, but it has different data depending on which intents are used: https://github.com/discord/discord-api-docs/pull/2855/files#diff-c24665b017972d8f7c266214d30655ea5e105826ef212048f9460632dd61e3dfR835

Reply:
maybe add asterisks and add a link to that section, people seem to often get confused by this